scripts/make_zephyr_sdk.sh

```diff
@@ -129,6 +130,13 @@ if [ -n "$file_gcc_riscv32" ]; then
   echo "echo \"\"" >>$setup
 fi
 
+if [ -n "$file_gcc_x86_64" ]; then
+  echo "tar -C \$target_sdk_dir -xf ./$file_gcc_x86_64 > /dev/null &" >> $setup
```

Comment:
can you update '-xf' to '-jxf`

Reply:
On both? or just x86_64?

Reply:
Just x86_64, I updated the others upstream already.

Reply:
Done.